### ISSUE TITLE
Toxic Exposure 1.0 Run validator against production data

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#88063]
## Issue Description
Submission data that would have previously passed primary path, also passes LH primary path. This is for a new dataset. 

---
## Tasks
- [x] Rerun old dataset to understand if the previous failures are now passing (ask Aurora for list)
- [x] Pick a day, run the following rake task:
bundle exec rake form526:lh_validate[local,<start_date>,<end_date>]
^ running this will tell us about any validation errors that occurred, including transformer errors and validation errors
- [x] Record/note any errors that are launch blocking, and create tickets for these

## Acceptance Criteria
- [x] Identified new issues categorized/fixed/resolved.

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by pacerwow** [comment]
May need to go into Argo to understand errors in detail, if there are issues.

**Comment by aurora-a-k-a-lightning** [comment]
Unique Validation Issues found:
1. Treatments/Treated Disability Names: The treated disability must match a disability listed above.
-- severe formatting issues
-- ticket to investigate, but also bring up with LH since these passed in EVSS
2. Separation Location: If Active Duty End Date is in the future a Separation Location Code is required.
-- we looked into this a bit and saw that Data Transformation #2 (saved_claim to EVSS format in the Form526Submission record) may not be capturing the Separation Location code properly
-- creating ticket
-- however, these are passing with the EVSS implementation. sending to LH for guidance
3. Mailing Address/City: The property 
/veteranIdentification/mailingAddress/city did 
not match the following requirements: 
{\"description\"=>\"City for the Veteran's 
current mailing 
address.\", \"type\"=>\"string\", \"pattern\"=>\"^
$|^([-a-zA-Z'.' ])+$\"
-- example: "29 Palms"
-- we brought this up to LH before with the Treatment Center city names. Now we need this on Mailing Address!
4. Direct Deposit: The property /directDeposit/accountNumber 
did not match the following requirements: 
{\"description\"=>\"Account number for the 
direct deposit.\", \"pattern\"=>\"^(?:[a-zA-Z0-9-]
{4,1000})?$\"
-- we have a ticket to address this on our side. not sure if LH got to it yet on their side
--- however, we saw some data samples with spaces in between the numbers such as "123456789 1234", which we can't address on our side. sending to LH for guidance
